apps/site/next-data/blogData.ts

```diff
@@ -28,7 +28,7 @@ const getBlogData = (cat: string, page?: number): Promise<BlogPostsRSC> => {
   // When we're on RSC with Server capabilities we prefer using Next.js Data Fetching
   // as this will load cached data from the server instead of generating data on the fly
   // this is extremely useful for ISR and SSG as it will not generate this data on every request
-  return fetch(fetchURL, { cache: 'force-cache' })
+  return fetch(fetchURL)
```

Comment:
Should we remove the other fetch() cache statements, I wonder?

Reply:
Yes. Linting was taking forever, so I figured I'd do one to make sure it worked. I'm linting the others (I might just --no-verify at this point)

Reply:
Since the pages themselves are cached we really don't need'em

Reply:
I feel like we should keep it uniform.

Reply:
the comment should likely be updated too once we stabilize

Reply:
I've updated it. Feel free to push to this branch if any of you want to change anything. You know more about caching in this codebase than I.